editor/components/block-preview/index.js

```diff
@@ -6,11 +6,13 @@ import { noop } from 'lodash';
 /**
  * WordPress dependencies
  */
+import { Component } from '@wordpress/element';
 import { createBlock, BlockEdit } from '@wordpress/blocks';
 
 /**
  * Internal dependencies
  */
+import { createInnerBlockList } from '../block-list/utils';
```

Comment:
I don't love that we reach into the local utilities of a separate component. Seems like if we need it in both places, might be best to extract up a level to a common place.

Reply:
👌 moved in e99f333aaeb2626b40f4c03c6fb8d5996443d50c.